tap_s3_csv/s3.py

```diff
@@ -395,7 +414,7 @@ def get_input_files_for_table(config, table_spec, modified_since=None):
     matched_files_count = 0
     unmatched_files_count = 0
     max_files_before_log = 30000
-    for s3_object in list_files_in_bucket(bucket, table_spec.get('search_prefix')):
+    for s3_object in list_files_in_bucket(config, table_spec.get('search_prefix')):
```

Comment:
Why do we remove the bucket?

Reply:
We changed the `list_files_in_bucket` function by replacing `bucket` with `config` because the function reads timeout from config so need the whole config. Also, a `bucket` is one field of the `config` so we can also utilize `bucket` from `config` at required places.